product.json

```diff
@@ -30,7 +30,7 @@
 	"win32TunnelServiceMutex": "positron-tunnelservice",
 	"win32TunnelMutex": "positrontunnel",
 	"darwinBundleIdentifier": "com.rstudio.positron",
-	"linuxIconName": "com.visualstudio.code.oss",
+	"linuxIconName": "com.rstudio.positron",
```

Comment:
Any thoughts on `co.posit.positron`?

Reply:
Sure! Do you think I should also change `darwinBundleIdentifier`